### ISSUE TITLE
Allure support for Jest Testing Framework

### ISSUE THREAD

**Issue by satyabratasaha-me** [allure-framework/allure2#686]
[//]: # (
. Note: for support questions, please use Stackoverflow or Gitter**. 
. This repository's issues are reserved for feature requests and bug reports.
.
. In case of any problems with Allure Jenkins plugin** please use the following repository 
. to create an issue: https://github.com/jenkinsci/allure-plugin/issues
.
. Make sure you have a clear name for your issue. The name should start with a capital 
. letter and no dot is required in the end of the sentence. An example of good issue names:
.
. - The report is broken in IE11
. - Add an ability to disable default plugins
. - Support emoji in test descriptions
)

#### I'm submitting a ... 
  - [ ] bug report
  - [ ] feature request
  - [ ] support request => Please do not submit support request here, see note at the top of this template.

#### Please tell us about your environment:

| Allure version | 2.2.0 |
| --- | --- |
| Test framework | testng@6.8 |
| Allure adaptor | allure-testng@2.0-BETA11 |
| Generate report using | allure-maven@2.18 |

#### Other information 

[//]: # (
. e.g. detailed explanation, stacktraces, related issues, suggestions 
. how to fix, links for us to have more context, eg. Stackoverflow, Gitter etc
)

**Comment by just-boris** [comment]
Jest's reporter api: http://facebook.github.io/jest/docs/en/configuration.html#reporters-array-modulename-modulename-options


@satyabratasaha-me very much unlikely that anyone from Allure team will implement this. But contributions are always welcome.

**Comment by baev** [comment]
Yeah, the issue is low priority but you can sponsor it. For more details please contact mailto:allure@qameta.io

**Comment by zaqqaz** [comment]
@satyabratasaha-me, @baev we did jest-allure reporting and you can check it here https://github.com/zaqqaz/jest-allure
